goalgorilla/open_social
# #2915492 - Some elements do not follow colour configuration

Currently, some elements do not follow colour configurations. They are:
- Footer
- Enrolled label
- 2ndary menu for user profile, group and event
- Comment button in activity stream item of 'someone commented on a topic/event'

How to test:
- Go to yoursite.com/admin/appearance/settings/socialblue
- Change the color values in the form
- Check the right colours are set in the above mentioned blocks/places.

https://www.drupal.org/node/2915492
